src/views/asset-view/asset-view.tsx

```diff
@@ -32,16 +35,17 @@ export const AssetView = (): JSX.Element => {
     );
   }
 
+  const showTenureInformation = isDwellingOrLettableNonDwelling(asset);
```

Comment:
smart

Reply:
Actually, I should refactor this. I repeat the line too many times